stunnel/init.sls

```diff
@@ -21,18 +21,27 @@ stunnel_package:
     - user: {{ stunnel.root_user }}
     - group: {{ stunnel.group }}
     - mode: 750
+{%- if stunnel.lookup.clean_conf_dir %}
```

Comment:
I don't think you need `.lookup.` here, the `stunnel:lookup` pillar is merged into the `stunnel` variable in `stunnel/map.jinja`

Reply:
also, nothing in the formula populates the `conf.d` folder, so I don't see why the formula should purge it